packages/react-devtools-extensions/src/__tests__/updateMockSourceMaps.js

```diff
@@ -63,6 +68,15 @@ function compile(fileName) {
     'utf8',
   );
 
+  // Generate compiled output with external inline base64 source maps
```

Comment:
Oops this looks like a clowny comment I wrote that should just say "inline base64" rather than "external inline base64"